tomls/omnibus-mainnet/cannon-permissions.toml

```diff
@@ -0,0 +1,529 @@
+        "0xEb3107117FEAd7de89Cd14D463D340A2E6917769",
+        "<%= settings.deployer %>",
+    ],
+    [
+        "0x6cd3f878852769e04A723A5f66CA7DD4d9E38A6C",
+        "<%= settings.deployer %>",
+    ],
+]
+
+[invoke.authorizedPublishers_synthetix_rewards_dist_ext]
+target = ["cannonRegistry.Proxy"]
+fromCall.func = "getPackageOwner"
+fromCall.args = ["<%= formatBytes32String('synthetix-rewards-dist-ext') %>"]
+func = "setAdditionalPublishers"
+args = [
+    "<%= formatBytes32String('synthetix-rewards-dist-ext') %>",
+    [
+        "0xEb3107117FEAd7de89Cd14D463D340A2E6917769",
+        "<%= settings.deployer %>",
+    ],
+    [
+        "0x6cd3f878852769e04A723A5f66CA7DD4d9E38A6C",
+        "<%= settings.deployer %>",
+    ],
+]
+
+[invoke.authorizedPublishers_fake_susde_token]
+target = ["cannonRegistry.Proxy"]
+fromCall.func = "getPackageOwner"
+fromCall.args = ["<%= formatBytes32String('fake-susde-token') %>"]
+func = "setAdditionalPublishers"
+args = [
+    "<%= formatBytes32String('fake-susde-token') %>",
+    [
+        "0xEb3107117FEAd7de89Cd14D463D340A2E6917769",
+        "<%= settings.deployer %>",
+    ],
+    [
+        "0x6cd3f878852769e04A723A5f66CA7DD4d9E38A6C",
+        "<%= settings.deployer %>",
+    ],
+]
+
+[invoke.authorizedPublishers_fake_usde_token]
+target = ["cannonRegistry.Proxy"]
+fromCall.func = "getPackageOwner"
+fromCall.args = ["<%= formatBytes32String('fake-usde-token') %>"]
+func = "setAdditionalPublishers"
+args = [
+    "<%= formatBytes32String('fake-usde-token') %>",
+    [
+        "0xEb3107117FEAd7de89Cd14D463D340A2E6917769",
+        "<%= settings.deployer %>",
+    ],
+    [
+        "0x6cd3f878852769e04A723A5f66CA7DD4d9E38A6C",
+        "<%= settings.deployer %>",
+    ],
+]
+
+[invoke.authorizedPublishers_fake_wsteth_token]
+target = ["cannonRegistry.Proxy"]
+fromCall.func = "getPackageOwner"
+fromCall.args = ["<%= formatBytes32String('fake-wsteth-token') %>"]
+func = "setAdditionalPublishers"
+args = [
+    "<%= formatBytes32String('fake-wsteth-token') %>",
+    [
+        "0xEb3107117FEAd7de89Cd14D463D340A2E6917769",
+        "<%= settings.deployer %>",
+    ],
+    [
+        "0x6cd3f878852769e04A723A5f66CA7DD4d9E38A6C",
+        "<%= settings.deployer %>",
+    ],
+]
+
+[invoke.authorizedPublishers_synthetix_snapshot_rewards]
+target = ["cannonRegistry.Proxy"]
+fromCall.func = "getPackageOwner"
+fromCall.args = ["<%= formatBytes32String('synthetix-snapshot-rewards') %>"]
+func = "setAdditionalPublishers"
+args = [
+    "<%= formatBytes32String('synthetix-snapshot-rewards') %>",
+    [
+        "0xEb3107117FEAd7de89Cd14D463D340A2E6917769",
+        "<%= settings.deployer %>",
+    ],
+    [
+        "0x6cd3f878852769e04A723A5f66CA7DD4d9E38A6C",
+        "<%= settings.deployer %>",
+    ],
+]
+
+[invoke.authorizedPublishers_chainlink]
+target = ["cannonRegistry.Proxy"]
+fromCall.func = "getPackageOwner"
+fromCall.args = ["<%= formatBytes32String('chainlink') %>"]
+func = "setAdditionalPublishers"
```

Comment:
how about removing publishers?

Reply:
its a "setter" so this function replaces any old publishers that may have existed.

Reply:
aaah ok. `additional` word confused me.